### DISCUSSION TITLE
404 on all redirects

### DISCUSSION THREAD

**Discussion by praetoriusd** [YOURLS/YOURLS#2994] [Q&A]
I can't seem to get the redirects to work. I have triple checked both my config.php and my .htaccess against the wiki and everything looks the same to me. I am not using Nginx and have gone through the first steps of the troubleshooting to no avail. 

Is there something else I could be missing here? 

Any help or guidance would be greatly appreciated. 

EDIT: I think this is a server config issue.

**Comment by outlander65** [comment]
Check this out:
https://github.com/YOURLS/YOURLS/discussions/2998
mybe it helps.

**Reply by praetoriusd** [reply]
Thanks. I spoke to a colleague of mine and he helped, sort of. 

We got it to stop returning 404 errors, but now it just takes you to the shortened link instead of redirecting to the URL entered. So weird.
